internal/commands/push/command.go

```diff
@@ -164,47 +152,24 @@ func (cmd *Command) Handler(profile *cli.Profile, ui terminal.UI) error {
 		}
 		defer os.Remove(uploadPath) //nolint:errcheck
 
-		if err := cmd.realmClient.ImportDependencies(to.GroupID, to.AppID, uploadPath); err != nil {
+		if err := clients.Realm.ImportDependencies(to.GroupID, to.AppID, uploadPath); err != nil {
 			return err
 		}
 	}
 
+	ui.Print(terminal.NewTextLog("Successfully pushed app changes"))
 	return nil
 }
 
-// Feedback is the command feedback
-func (cmd *Command) Feedback(profile *cli.Profile, ui terminal.UI) error {
-	if cmd.outputs.noDiffs {
-		return ui.Print(terminal.NewTextLog("Deployed app is identical to proposed version, nothing to do"))
-	}
-	if cmd.outputs.diffRejected || cmd.outputs.draftKept {
-		return ui.Print(terminal.NewTextLog("No changes were pushed to your Realm application"))
-	}
-
-	if cmd.outputs.appCreated {
-		if cmd.inputs.DryRun {
-			return ui.Print(
-				terminal.NewTextLog("This is a new app. To create a new app, you must omit the 'dry-run' flag to proceed"),
-				terminal.NewSuggestedCommandsLog(cmd.commandString(true)),
-			)
-		}
-		return ui.Print(
-			terminal.NewTextLog("This is a new app. You must create a new app to proceed"),
-		)
-	}
-
-	return ui.Print(terminal.NewTextLog("Successfully pushed app changes"))
-}
-
 type namer interface{ Name() string }
 type locationer interface{ Location() realm.Location }
 type deploymentModeler interface{ DeploymentModel() realm.DeploymentModel }
 
-func (cmd *Command) createNewApp(ui terminal.UI, groupID string, appData interface{}) (realm.App, error) {
+func createNewApp(ui terminal.UI, realmClient realm.Client, appDirectory, groupID string, appData interface{}) (realm.App, bool, error) {
```

Comment:
I would rather throw a named error if the ui doesn't wish to proceed than add another return value of `bool`. How are we planning on handling the case where the user doesn't want the confirm prompt (ie: from a ci pipeline)?

Reply:
to bypass confirm prompts, the CLI (and this is existing behavior) exposes a `--yes, -y` flag (which is checked during `ui.Confirm(...)`, but is also exposed through `ui.AutoConfirm()`).

there's a smell here, that we need to pay attention to that detail two different ways, so i'm all ears if you have any thoughts.  but this behavior is something i more or less grabbed from the existing cli